syntax = "proto3";

service EchoService {
  rpc Echo(EchoRequest) returns (EchoReply) {}
}

message EchoRequest {
  string msg = 1;
}

message EchoReply {
  string msg = 1;
}
